katas/move-zeros/move-zeros.js

```diff
@@ -1,5 +1,16 @@
-function moveZeros() {
-
+function move_zeros(arrNum, isRight = true){
+  const zeroArray = [];
+  let i = 0;
+  while (i<arrNum.length){
```

Comment:
not sure, because we will change `i` only in case if array item is not equal to zero